Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM centos:7
 MAINTAINER Alvaro Vega <alvaro.vegagarcia@telefonica.com>
 
 ENV ORCHESTRATOR_USER orchestrator
-ENV ORCHESTRATOR_VERSION 1.6.0
+ENV ORCHESTRATOR_VERSION 1.7.0
```

Comment:
ORCHESTRATOR_VERSION should be also adjusted in the release/1.7.0 branch (and the tag 1.7.0 moved to the new HEAD of release/1.7.0)?

Reply:
NTC